profile/README.md

```diff
@@ -14,3 +14,149 @@ The OSTIS technology is implemented as a network of ostis-systems, which is part
+- The development of such systems costs enormous resources.
+
+It's not hard to notice that the main problem is expandability of such systems. Thus, we need a framework to design intelligent systems that are compatible with each other, because interaction between intelligent systems is one of the ways to enable intelligent systems to solve new classes of problems. OSTIS is a technology to create **semantically compatible** intelligent systems.
+
+## Principles
+
+### **SC-code**
+
+The OSTIS Technology is based on a universal way to represent information, named SC-code (Semantic Computer Code). The SC-code is based on the basic formalisms of discrete mathematics (set theory and graph theory).
+
+That ensures:
+
+- **Universality** and **uniformity** of information representation (any information can be represented in the same way).
+
+- The convenience of machine processing and human perception.
+
+The SC-code is a **standard** for universal sense representation of information in the memory of computer systems.
+
+SC-code features:
+
+- The SC-code is non-linear, so it can be used to encode information in the memory of semantic associative computers.
+- The basic alphabet of the SC-code consists of only 5 elements, on the basis of which more and more complex structures are built.
+- You can use the SC-code to represent not only the knowledge of the system but also the problem-solving models and system interface.
+- The SC-code is abstract language and can be visualized in different forms. Currently, there are several forms of the SC-code: SCg-, SCn-, SCs-code.
+
+![Simple logic rule in SCg](docs/scg-examples/scg-logic-rule.jpeg)
+
+*Simple logic rule represented in the SCg-code: "If X is a bird, then X is an animal"*
+
+### **Architecture**
+
+Every OSTIS-based system (referenced as ostis-system down the line for convenience) consists of the following components:
+
+- The OSTIS knowledge base: can describe any kind of knowledge, easy to supplement with new knowledge types.
+
+- The OSTIS problem solver: based on a multi-agent approach, allows integrating and combining any problem-solving models.
+
+- The interface of the ostis-system: a special kind of knowledge base and problem solver, i.e. is also described by means of the SC-code.
+
+## Advantages
+
+- **Any** problem-solving model or knowledge can be seamlessly integrated into **any** ostis-system without **any** overhead costs (by the plug & play principle). Thus, it does not matter what the system can do at the moment: it can always be retrained to solve another problem
+
+- Existing components of ostis-systems are universal (they can be used in completely different systems) and are compatible with each other. Thus, it is possible to accumulate a library of components and reuse components, greatly reducing the system evolution laboriousness. Аt the moment, the universal part (core) of knowledge bases can reduce the development time of the knowledge base components by 40-60%.
```

Comment:
how did you calculated 40-60%?

Reply:
it was here https://github.com/ostis-ai/ostis-project